webapp/src/contexts/ConferenceContext/ConferenceReducer.ts

```diff
@@ -13,6 +13,9 @@ export const ConferenceReducer = (prevState: ConferenceState, action: Conference
     case ConferenceActionType.Connecting: {
       return {
         ...prevState,
+        audioMuted: false,
```

Comment:
Just to understand: why are these values always false in the connecting state?

Reply:
It's to reset the state of the buttons when we disconnect and make a second call. But I think that maybe we could move them to `Connected`. Maybe it makes more sense there:

I mean these values:
```
  audioMuted: false,
  videoMuted: false,
  presenting: false,
```

Reply:
I moved it to the Connected state.